specs/commandline/certificate.md

```diff
@@ -130,7 +130,6 @@ Flags:
   -b, --bits int      RSA key bits (default 2048)
   -d, --default       mark as default
   -h, --help          help for generate-test
-  -n, --name string   key and certificate name
       --trust         add the generated certificate to the trust store
```

Comment:
Remove `--trust` flag for generate-test command, and the generated certificate should be added into trust store by default, otherwise user need to get the certificate file path and use `notation cert add` to add the certificate file to trust store, which is not cumbersome.

Reply:
Good point!